bundles/tools.vitruv.applications.util.temporary/src/tools/vitruv/applications/util/temporary/uml/UmlClassifierAndPackageUtil.xtend

```diff
@@ -36,14 +36,15 @@ class UmlClassifierAndPackageUtil {
 
     /**
      * Searches and retrieves the UML package in the UML model that has an equal name as the given package name.
+     * The routines ignores the casing of the package names.
      * If there is more than one package with the given name, an {@link IllegalStateException} is thrown.
      * @param umlModel the UML model Model in which the UML packages should be searched
      * @param packageName the package name for which a fitting UML package should be retrieved
      * @return the UML package or null if none could be found
      */
     def static Package findUmlPackage(Model umlModel, String packageName) {
         val Set<Package> allPackages = umlModel.eAllContents.filter(Package).toSet
-        val packages = allPackages.filter[name == packageName]
+        val packages = allPackages.filter[name?.toLowerCase == packageName?.toLowerCase]
```

Comment:
Is there any reason to not use the `isPackageFor` method here too? I checked all calls of the method, we would always have a model element to pass instead of just the package name string.

Reply:
I think I missed that one (maybe I did not look for `toLowerCase` conversions), at least I cannot remember a good reason for not using `isPackageFor` here.
Thank you for already checking that we can provide a model element here to validate the package against, then it should be easy to fix. Otherwise it would even make sense to have some more general validation method for package names that ignores their capitalization to be also used for the existing `isPackageFor` method.

Reply:
I have taken a look at the issue. The problem is that this method is generic for UML and used for elements from both PCM and Java. So the name of the elements must somehow be extracted before passing them to this method. Since we actually have to make an extension for not only checking for the name of the package but also for its namespaces (see #177), I would propose to keep this implementation and open an issue to fix the validation along with an extension for checking the namespace as well.

Reply:
I've documented that in #178.